addon/adapters/cloud-firestore.js

```diff
@@ -13,10 +13,6 @@ import RealtimeTracker from 'ember-cloud-firestore-adapter/utils/realtime-tracke
 export default class CloudFirestoreAdapter extends Adapter {
   @service firebase;
 
-  firestoreSettings = null;
-
-  referenceKeyName = 'referenceTo';
-
```

Comment:
I saw your comment in one of my PRs regarding the bug here. The solution would be to convert back the `constructor` into an `init` hook again. Thanks for catching this, I'll provide a fix for this in a while.